patches/twig-twig-src-tokenparser-macrotokenparser-php.patch

```diff
@@ -0,0 +1,19 @@
+--- /dev/null
++++ ../src/TokenParser/MacroTokenParser.php
+@@ -11,6 +11,7 @@
+ 
+ namespace Twig\TokenParser;
+ 
++use Shopware\Core\Framework\Adapter\Twig\Node\MacroOverrideNode;
+ use Twig\Error\SyntaxError;
+ use Twig\Node\BodyNode;
+ use Twig\Node\EmptyNode;
+@@ -54,7 +55,7 @@
+         $this->parser->popLocalScope();
+         $stream->expect(Token::BLOCK_END_TYPE);
+ 
+-        $this->parser->setMacro($name, new MacroNode($name, new BodyNode([$body]), $arguments, $lineno));
++        $this->parser->setMacro($name, new MacroOverrideNode($name, new BodyNode([$body]), $arguments, $lineno));
```

Comment:
there currently is no good way to do that, other than copying the whole TokenParser as well

But maybe we can update it and make it available under sw_macro? And the return then only works there? That would of course be a break for apps